pipelines/build-release.yml

```diff
@@ -13,7 +13,7 @@ resource_types:
   - name: pull-request
     type: docker-image
     source:
-      repository: governmentpaas/pullrequest-resource
+      repository: jtarchie/pr
```

Comment:
Regarding permanency there are a few stats which indicate it is here to stay:

* Over 1 million pulls on Docker hub.
* Twenty versions released so far.
* Actively maintained on Github: 13 contributors and commits regularly.

However, nothing is permanent and there is nothing stopping the author taking it down. Worst case scenario it is taken down and we are faced with a mild inconvenience.

Regarding trust, the author's Docker hub password could be 'password' for all I know, but from my interactions with him he seems like a very competent developer.